.github/workflows/test.yaml

```diff
@@ -23,5 +23,8 @@ jobs:
       - name: Install Dependencies
         run: npm ci
 
+      - name: Set environment variable
+        run: export CHECK_CONTRACT_SIZE=true
```

Comment:
Would it make more sense to set the env var only for the following test command instead of exporting it into the env? Something like
```
      - name: Test
        run: CHECK_CONTRACT_SIZE=true npm run test-evm-versions
```

Reply:
true, will change that in other repos as well